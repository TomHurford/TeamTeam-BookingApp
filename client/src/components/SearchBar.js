import React, {useState} from "react";

//Creating a search bar component

function SearchBar(){
    const [input, setInput] = useState('');
    const handleInput = (e) => {
        setInput(e.target.value);
    }
    
    return(
       <form>
              <input type="text" value={input} onChange={handleInput} placeholder = "Search for events..."/>
       </form>
    )
    
}
export default SearchBar;