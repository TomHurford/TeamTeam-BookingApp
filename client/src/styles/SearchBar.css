.searchBarContainer{
    position: relative;
    display: inline-block;
    justify-content: center;
    width: 1000px;
    left: 20%;
}

input{
    display: flex;
    font-size: 25px;
    border-radius: 40px;
    border: #999;
    height: 30px;
    padding: 20px;
    padding-left: 50px;
    width: 200px;
    
}

.searchIcon{
    position: fixed;
    right: 955px;
    width: 35px;
    height: 30px;
    margin-top: 5px;
}