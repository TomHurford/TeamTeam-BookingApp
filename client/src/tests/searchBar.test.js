import React from 'react';
import renderer from 'react-test-renderer';
import SearchBar from '../components/searchBar';

test('searchBar snapshot test', () => {
  const component = renderer.create(<SearchBar />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});