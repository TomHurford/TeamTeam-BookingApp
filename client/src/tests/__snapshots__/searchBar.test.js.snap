// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`searchBar snapshot test 1`] = `
<input
  className="form-control my-4"
  name="query"
  onChange={[Function]}
  placeholder="Search for society..."
  type="text"
/>
`;
